### ISSUE TITLE
Perf: VK_IMAGE_USAGE_TRANSFER_DST_BIT should not be enabled for render targets 

### ISSUE THREAD

**Issue by hrydgard** [hrydgard/ppsspp#12442]
Got information that enabling VK_IMAGE_USAGE_TRANSFER_DST_BIT on render targets can somewhat reduce performance on some hardware, and that it's usually better to just draw a textured quad than to use vkCmdBlitImage, where possible (color buffers).

Not sure if making a change like that would really have much noticable impact though...

**Comment by fahadrashid7705** [comment]
only for vulkan ?

**Comment by hrydgard** [comment]
This is relevant only for Vulkan, and probably not very important. Just some information I got.

**Comment by Gamemulatorer** [comment]
This is about performance? maybe this is the cause of https://github.com/hrydgard/ppsspp/issues/15136? Just guessing 😅

**Comment by hrydgard** [comment]
Nah, irrelevant for sure. This one is likely a very small penalty anyway and barely worth doing.

**Comment by Gamemulatorer** [comment]
Does ppsspp enable support
vkCmdCopyImageToBuffer on Qualcomm?

**Comment by hrydgard** [comment]
yeah, all hardware supports that function. it's mainly used for readbacks from GPU to CPU.
